addons/website/views/website_pages_views.xml

```diff
@@ -98,7 +96,6 @@
     <field name="priority">99</field>
     <field name="arch" type="xml">
         <list js_class="website_pages_list" type="object" action="open_website_url" multi_edit="1" default_order="url asc, id asc">
-            <field name="is_homepage" column_invisible="True"/>
```

Comment:
This one is needed to display the small house icon (for home) in the pages settings (Site > Pages) :slightly_smiling_face: 
<img width="311" height="165" alt="image" src="https://github.com/user-attachments/assets/f1f260a3-3371-481d-ac7a-fad8098c9b7e" />